src/GitHub.InlineReviews/Services/ICommentService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace GitHub.InlineReviews.Services
+{
+    public interface ICommentService
```

Comment:
Would be good to get some XML docs on this service to indicate where/for what it's used.